controllers/kuadrant_controller.go

```diff
@@ -142,6 +141,10 @@ func (r *KuadrantReconciler) Reconcile(eventCtx context.Context, req ctrl.Reques
 		}
 	}
 
+	if gwErr := r.reconcileClusterGateways(ctx, kObj); gwErr != nil {
```

Comment:
What drove this? This is new, right?

Reply:
Yes, this is basically the key step to communicate the Kuadrant instance namespace to the policies. It will annotate the Gateways with its namespace, thus the policies, who knows about the GW or at least the HTTPRoute pointing to the Gateway will have access to this annotation